theano/tensor/elemwise.py

```diff
@@ -492,6 +493,8 @@ class Elemwise(OpenMPOp):
 
     """
 
+    __props__ = ("scalar_op", "inplace_pattern")
```

Comment:
No. This is a special case. OpenMP do not change the result. So if we have 2 nodes in the graph that have the same inputs and are only different by openmp, we want them to be merged. If we add openmp here, they won't be merged. We don't know which one will be kept, but that is less troublesome then not merging them.